global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
-        "version": "6.0.100",
-        "rollForward": "latestFeature"
+        "version": "6.0.*",
+        "rollForward": "latestPatch"
```

Comment:
@taooceros what's this change for?

Reply:
Oh it shouldn't be there (I am testing different version of dotnet and see which one can help with the bug. However it is not needed because workaround is found)...Though it makes sense to apply the latest patch since that generally includes some security fixes.